modules/react/switch/lib/Switch.tsx

```diff
@@ -70,14 +70,14 @@ const switchInputStencil = createStencil({
       },
       '&:disabled, &.disabled': {
         '& ~ div:first-of-type': {
-          backgroundColor: brand.primary.light,
```

Comment:
Talked to design, instead of using backgroundColor to show something disabled, we should use opacity instead. This is a visual breaking change.